tests/test_domain_c.py

```diff
@@ -590,3 +591,13 @@ def test_cvar(app):
     domain = app.env.get_domain('c')
     entry = domain.objects.get('PyClass_Type')
     assert entry == ('index', 'c.PyClass_Type', 'var')
+
+
+def test_noindexentry(app):
+    text = (".. c:function:: void f()\n"
+            ".. c:function:: void g()\n"
+	        "   :noindexentry:\n")
```

Comment:
TAB char!

Reply:
Oops, my IDE was supposed to to hide my tabbing habits. Though, how come flake8 didn't catch it? It seems to ignore all the test files.